pages/common/black.md

```diff
@@ -22,3 +22,7 @@
 - Auto-format a file or directory emitting exclusively error messages to stderr:
 
 `black --quiet {{path/to/file_or_directory}}`
+
+- Auto-format a file or directory without replacing single quotes with double quotes:
+
+`black -S {{path/to/file_or_directory}}`
```

Comment:
We prefer long arguments when are available:
```suggestion
`black --skip-string-normalization {{path/to/file_or_directory}}`
```

Reply:
@navarroaxel Perfect. I do, too. Should I update the existing short argument, etc. `black -c => black --code`?

Reply:
Done in 49b7216.